package.json

```diff
@@ -10,53 +10,53 @@
     }
   ],
   "dependencies": {
-    "@styled-system/css": "5.1.4",
+    "@styled-system/css": "5.1.5",
     "@types/color": "^3.0.1",
-    "@types/styled-components": "^4.4.2",
-    "@types/styled-system": "^5.1.5",
+    "@types/styled-components": "^4.4.3",
+    "@types/styled-system": "^5.1.6",
     "@types/styled-system__css": "^5.0.4",
     "color": "3.1.2",
     "prop-types": "^15.7.2",
     "react-spring": "8.0.27",
-    "styled-system": "5.1.4",
+    "styled-system": "5.1.5",
     "use-dark-mode": "2.3.1",
     "use-events": "^1.4.1"
   },
   "devDependencies": {
-    "@babel/cli": "7.8.3",
-    "@babel/core": "7.8.3",
+    "@babel/cli": "7.8.4",
+    "@babel/core": "7.8.4",
     "@babel/plugin-proposal-class-properties": "7.8.3",
     "@babel/plugin-proposal-object-rest-spread": "7.8.3",
     "@babel/plugin-transform-object-assign": "7.8.3",
     "@babel/plugin-transform-runtime": "7.8.3",
-    "@babel/preset-env": "7.8.3",
+    "@babel/preset-env": "7.8.4",
     "@babel/preset-react": "7.8.3",
     "@babel/preset-typescript": "^7.8.3",
     "@blockstack/prettier-config": "^0.0.5",
-    "@typescript-eslint/eslint-plugin": "^2.18.0",
-    "@typescript-eslint/parser": "^2.18.0",
+    "@typescript-eslint/eslint-plugin": "^2.20.0",
+    "@typescript-eslint/parser": "^2.20.0",
     "babel-plugin-module-resolver": "4.0.0",
     "babel-plugin-react-remove-properties": "0.3.0",
     "babel-plugin-transform-react-remove-prop-types": "0.4.24",
     "bundlesize": "^0.18.0",
     "cross-env": "7.0.0",
-    "dtslint": "^2.0.5",
+    "dtslint": "^3.0.0",
```

Comment:
Looks like we aren't using this, and we shouldn't need it, since it's mainly for writing `.d.ts` files by hand from JS.